streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractStateManager.java

```diff
@@ -45,7 +45,6 @@
     OffsetCheckpoint checkpoint;
 
     final Map<TopicPartition, Long> checkpointableOffsets = new HashMap<>();
-    final Map<String, StateStore> stores = new LinkedHashMap<>();
```

Comment:
This was only used in ProcessorStateManager, so I moved it there and made it private.

Reply:
+100